app/templates/components/labs-map.hbs

```diff
@@ -0,0 +1,47 @@
+{{!--
+  This template overrides the template for an addon, ember-mapbox-composer.
+  e-mapbox-composer uses private APIs in e-mapbox-gl. Those changed, so we need to
```

Comment:
does the new interface defined here also use private APIs?

Reply:
hmm basically the old version of this component used some private apis in the addon ember-mapbox-gl. I updated ember-mapbox-gl, but it's private apis changed, as they do. so, we still use those private apis, but that should be addressed later.